pylatex/document.py

```diff
@@ -26,7 +27,9 @@ class Document(Environment):
 
     def __init__(self, default_filepath='default_filepath', *,
                  documentclass='article', fontenc='T1', inputenc='utf8',
-                 lmodern=True, textcomp=True, data=None):
+                 lmodern=True, textcomp=True, page_numbers=True, indent=False,
```

Comment:
- Could you leave the data argument at the end?
- add document_options after document_class
- and fontsize after inputenc.
- do the same ordering for the arguments in the docstring.